pocs/camera/camera.py

```diff
@@ -300,6 +300,8 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
                                                                          filename,
                                                                          **kwargs)
 
+        # pop exptime from kwarg as its now in exptime
+        exptime = kwargs.pop('exptime', observation.exptime.value)
```

Comment:
Sorry for being so late to the game. `exptime` here is a keyword argument that can be used to override the `observation.exptime.value` and `take_observation` is the only consumer of that, so it probably just shouldn't be hidden away in `kwargs` but as an actual documented parameter.